scripts/github_scraper.py

```diff
@@ -36,11 +37,11 @@ def get_github_info(target_url):
     return name, description, stars
 
 
-def return_details(target_urls):
+def return_details(target_urls:List[str])->Dict[str, Dict[str, str]]:
```

Comment:
```suggestion
def return_details(target_urls:list[str])->dict[str, dict[str, str]]:
```
Like this, for example.

Reply:
Thank you for pointing this out! Will work on it👍